### PR TITLE
Feat/sequel compilers

### PR DESCRIPTION
**Pull request by nvh0412**
### Motivation
Resolved https://github.com/Shopify/tapioca/issues/177

### Implementation
Based on our current ActiveRecord Implementation, we implement the same way for Sequel Association and Column.

### CODE REVIEW

File: lib/tapioca/generator.rb [hunk 1]
```diff
@@ -223,13 +223,15 @@ def say_error(message = "", *color)
 
     sig { params(eager_load: T::Boolean).void }
     def load_application(eager_load:)
-      say("Loading Rails application... ")
+      say("Loading Rails application and Non-Rails application")
```
Comment (Morriar) [inline lib/tapioca/generator.rb:226]:
```suggestion
      say("Loading application...")
```

File: lib/tapioca/compilers/dsl/sequel_columns.rb [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+        end
+
+        sig do
+          params(
+            constant: T.class_of(Sequel::Model),
+            column_name: String
+          ).returns([String, String])
+        end
+        def type_for(constant, column_name)
+          return ["T.untyped", "T.untyped"] if do_not_generate_strong_types?(constant)
```
Comment (Morriar) [inline lib/tapioca/compilers/dsl/sequel_columns.rb:169]: Do you also need the `StrongTypeGeneration` pattern for sequel or is it to mirror what we do in the Active Record generators?

We might want the reverse choice, something like `UnsafeTypeGeneration` as to encourage safe typing by default.

**Review by Morriar — CHANGES_REQUESTED** [review]
I'm not familiar with sequel but the changes look good 👍 

Can you add some tests for your compilers, like it's done for [Active Record](https://github.com/Shopify/tapioca/blob/master/spec/tapioca/compilers/dsl/active_record_columns_spec.rb) please?

Thanks for you pull-request!

**Comment by rafaelfranca** [comment]
I don't think we should maintain Sequel compilers in tapioca. Our compilers system make it possible to people to write their own compilers, so I think a compiler to Sequel should be in its own gem, maintained by people that uses Sequel.

**Comment by nvh0412** [comment]
It makes sense, @rafaelfranca. I realized that Sequel compilers should be maintained by people that use Sequel instead of putting it into this Gem when I was implementing this stuff. Let me close this PR. Thank you, @Morriar too

**Pull request closed without merging**